conda-recipe/meta.yaml

```diff
@@ -58,15 +61,15 @@ requirements:
     - cython
     - jinja2
     - pybind11
-    - numpy {{ numpy }}
+    - numpy
     - impi-devel  # [not win]
     # dal-devel pinning depends on the recipe location (repo or feedstock)
     - dal-devel
     # - dal-devel =={{ version }}
   run:
     - python
-    - {{ pin_compatible('numpy') }}
-    - dpcpp-cpp-rt  # [linux64]
+    - numpy
+    - scikit-learn
```

Comment:
Also, is scikit-learn needed in build deps or should it remain as test dep?

Reply:
`scikit-learn` is only run time dependency.